### PR TITLE
strategy/bollinger_bands

### PR DESCRIPTION
**Pull request by hemendra-sakpal-algobulls**

### CODE REVIEW

File: pyalgostrategypool/bollinger_bands.py [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+        """
+        return 'Bollinger Bands'
+
+    @staticmethod
+    def versions_supported():
+        """
+        Strategy should always support the latest engine version
+        Current version is 3.3.0
+        """
+        return [AlgoBullsEngineVersion.VERSION_3_3_0]
```
Comment (akhil-jain-algobulls) [inline pyalgostrategypool/bollinger_bands.py:51]: list is required?

File: pyalgostrategypool/bollinger_bands.py [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+        # Check entry/exit conditions
+        if (previous_candle['open'] <= lowerband_value or previous_candle['high'] <= lowerband_value or previous_candle['low'] <= lowerband_value or previous_candle['close'] <= lowerband_value) and \
+                (latest_candle['close'] > previous_candle['close']):
+            action = 'BUY'
+        elif (previous_candle['open'] >= upperband_value or previous_candle['high'] >= upperband_value or previous_candle['low'] >= upperband_value or previous_candle['close'] >= upperband_value) and \
+                (latest_candle['close'] < previous_candle['close']):
+            action = 'SELL'
+        else:
+            action = None
+        logger.info(f'Previous candle: {[previous_candle]}\n'
```
Comment (akhil-jain-algobulls) [inline pyalgostrategypool/bollinger_bands.py:81]: why do we have [ ] for previous candle ?

dont print the complete candle

print ohlc of previous candle in one line, then `\n`, then latest close upper band and lower band in second line, then `\n`, then action in the third line

File: pyalgostrategypool/bollinger_bands.py (hunk 1 above)
Comment (hemendra-sakpal-algobulls) [reply pyalgostrategypool/bollinger_bands.py:54]: List removed

File: pyalgostrategypool/bollinger_bands.py (hunk 2 above)
Comment (hemendra-sakpal-algobulls) [reply pyalgostrategypool/bollinger_bands.py]: Done

**Pull request closed without merging**